src/bun.js/webcore/response.zig

```diff
@@ -1911,7 +1911,23 @@ pub const Fetch = struct {
                     }
                 } else {
                     method = request.method;
+
+                    if (request.body.value == .Locked) {
+                        if (request.body.value.Locked.readable) |stream| {
+                            if (stream.isDisturbed(globalThis)) {
```

Comment:
should we also check isDisturbed in other places we call `useAsAnyBlob`? like: https://github.com/oven-sh/bun/blob/99aef8c217c783d5a23ed5911dca77678b6f1bde/src/bun.js/webcore/response.zig#L1996 https://github.com/oven-sh/bun/blob/99aef8c217c783d5a23ed5911dca77678b6f1bde/src/bun.js/webcore/response.zig#L1779